src/NotifyQueueConsumer/Queue/SqsAdapter.php

```diff
@@ -41,11 +41,17 @@ public function next(): ?SendToNotify
 
         $this->validateBody($body);
 
+        $message = $body['message'];
+
         return SendToNotify::fromArray([
             'id' => $raw['ReceiptHandle'],
-            'uuid' => $body['message']['uuid'],
-            'filename' => $body['message']['filename'],
-            'documentId' => $body['message']['documentId'],
+            'uuid' => $message['uuid'],
+            'filename' => $message['filename'],
+            'documentId' => $message['documentId'],
+            'documentType' => $message['documentType'] ?? null,
```

Comment:
Unless this is doing some PHP magic that I don't understand, I'm not sure I get why you are doing `?? null`. `??` is the null coalescing operator, so it takes the left hand value if not null and the right hand if it is null. So this is saying "if `$message['documentType']` is `null`, use `null` instead, which is the same as just using the left hand value in the first place.